### PR TITLE
Readme cleanup

### PR DESCRIPTION
**Pull request by brson**
Just a few things to make the README ever more perfect.

r? @steveklabnik

### CODE REVIEW

**Comment by brson** [comment]
[Rendered](https://github.com/brson/rust/tree/readme-cleanup)

File: README.md [hunk 1]
```diff
@@ -93,8 +88,9 @@ supported build environments that are most likely to work.
 Rust currently needs about 1.5 GiB of RAM to build without swapping; if it hits
 swap, it will take a very long time to build.
 
-There is a lot more documentation in the [wiki].
+There is more advice about [hacking on Rust][hack] on the [wiki].
 
+[hack]: https://github.com/rust-lang/rust/wiki/Note-getting-started-developing-Rust
```
Comment (steveklabnik) [inline README.md]: we want to kill the wiki, so this isn't good. maybe the CONTRIBUTING.md link is the righ tcall here?

File: README.md [hunk 2]
```diff
@@ -18,22 +20,15 @@ Read ["Installing Rust"][install] from [The Book][trpl].
...
-    You can either download a [tarball] or build directly from the [repo].
-
-    To build from the [tarball] do:
-
-        $ curl -O https://static.rust-lang.org/dist/rustc-nightly-src.tar.gz
-        $ tar -xzf rustc-nightly-src.tar.gz
-        $ cd rustc-nightly
-
-    Or to build from the [repo] do:
+2. Clone the [source] with git:
```
Comment (steveklabnik) [inline README.md]: this git should be in graves: `git`

File: README.md [hunk 3]
```diff
@@ -111,6 +107,14 @@ The Rust community congregates in a few places:
 [users.rust-lang.org]: http://users.rust-lang.org/
 [internals.rust-lang.org]: http://internals.rust-lang.org/
 
+Rust has an [IRC] culture and most real-time collaboration happens in a
+variety of channels on Mozilla's IRC network, irc.mozilla.org. The
+most popular channel is [#rust], a venue for general discussion about
+Rust, and a good place to ask for help,
+
+[IRC]: https://en.wikipedia.org/wiki/Internet_Relay_Chat
+[#rust]: https://chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust
```
Comment (steveklabnik) [inline README.md]: should we use an `irc://` link over a mibbit link? maybe not.

**Comment by steveklabnik** [comment]
This looks great in general, r=me after we address these few nits

**Comment by Manishearth** [comment]
Needs a rebase, sorry.

**Pull request merged**